### DISCUSSION TITLE
Valid solution not accepted

### DISCUSSION THREAD

**Discussion by gunnarbittersmann** [cssbattle/forum#95] [Bugs reports]
Using `<div class="svg">` should be totally valid, should it?
However, submitting is disabled: “SVGs are not allowed”
I ain’t using no SVG, you 🤬💣☠️ server! 🤣

**Comment by chinchang** [comment]
hahaha...it should be fixed definitely. But I am sure you can work around it until we fix it 😆

**Reply by gunnarbittersmann** [reply]
Yo. I’ve renamed my class. `<div class="script">` works. 🤪

**Reply by gunnarbittersmann** [reply]
Just for the record: I would never use neither `div` nor `class` in code-golfing. So shouldn’t you.
